index.html

```diff
@@ -23,26 +23,37 @@
 
       gtag("config", "G-L9EM4V5T72");
     </script>
+    <!-- end of Google Analytic -->
+
     <title>Novell's Portfolio</title>
   </head>
 
   <body>
     <!-- nav start -->
     <nav>
       <div class="logo">
-        <img src="IMG/my-logo.png" alt="Novell's Logo">
+        <img src="IMG/my-logo.png" alt="">
       </div>
-
       <ul>
-        <li><a href="#">Portfolio</a></li>
-        <li><a href="#">About</a></li>
-        <li><a href="#">Home</a></li>
+        <li>
+          <a href="#">Portfolio</a>
+        </li>
+        <li>
+          <a href="#">About</a>
+        </li>
+        <li>
+          <a href="#">Contact</a>
+        </li>
       </ul>
```

Comment:
- [ ] Currently when I click one option of the mobile menu it won't disappear kindly fix that and I suggest that you link each list element with a related section not using dead links.

Reply:
oh year thanks

Reply:
Well addressed